javascript/javascript3/week1/weatherApplcation/app.js

```diff
@@ -0,0 +1,119 @@
+//google api =AIzaSyCFaZoWcHNecGECjLPw9VuU7D89ARNieg0
+const ApiKey = '0904bdb6c689df374189c5ab06c21df6 '
+
+//get elements
+const button = document.getElementById('button')
+const input = document.getElementById('input')
+const main = document.querySelector('main')
+const iconElement = document.querySelector('.weather-icon img')
+
+const message = document.getElementById('msg')
+
+// Creat elements
+const cityName = document.createElement('h3')
+const temperature = document.createElement('div')
+const weatherType = document.createElement('div')
+const windSpeed = document.createElement('div')
+const clouds = document.createElement('div')
+const sunrise = document.createElement('div')
+const sunset = document.createElement('div')
+
+// Append elements
+main.appendChild(cityName)
+main.appendChild(temperature)
+main.appendChild(clouds)
+main.appendChild(weatherType)
+main.appendChild(windSpeed)
+main.appendChild(sunset)
+main.appendChild(sunrise)
+//google elemnents
+const searchElement = document.getElementById('map')
+const userLocation = document.getElementById('userLocation')
+const searchedCity = ''
+const latData = 0
+const lngData = 0
+
+// main function to get data from JSON
+const getWeather = function () {
+    let searchedCity = input.value
+    if (input.value == '') {
+        message.innerText = 'Please enter Valid city name'
+        message.style.backgroundColor = 'red'
+    } else {
+        message.style.visibility = 'hidden'
+        fetch(`https://api.openweathermap.org/data/2.5/weather?q=${searchedCity}&units=metric&appid=${ApiKey}`)
+            .then(Response => Response.json())
```

Comment:
Even though it is technically valid to name a variable with an uppercase letter, it is not a good idea.

See the [naming convention](https://github.com/HackYourFuture/fundamentals/blob/master/fundamentals/naming_conventions.md).

```suggestion
            .then(response => response.json())
```